src/components/Home/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styled from 'react-emotion';
```

Comment:
Using `emotion` instead of `styled-components`.
Because it has the same syntax as `styled-components` and more features are available.
https://github.com/emotion-js/emotion